src/Generation/GapicClientV2Generator.php

```diff
@@ -876,4 +876,14 @@ private function snippetPathForMethod(MethodDetails $method): string
 
         return "samples/{$version}{$emptyClientName}/{$methodName}.php";
     }
+
+    private function asyncReturnType(MethodDetails $method): string
+    {
+        $returnType = $this->ctx->type(Type::fromName(PromiseInterface::class))->type->name;
+        $nestedType = ($method->hasEmptyResponse) ?
+            'void' :
+            $this->ctx->type($method->methodReturnType)->type->name;
```

Comment:
nit: format it like this:
```suggestion
        $nestedType = ($method->hasEmptyResponse) 
            ? 'void'
            : $this->ctx->type($method->methodReturnType)->type->name;
```